authors/apps/articles/views.py

```diff
@@ -89,3 +86,123 @@ def delete(self, request, slug):
         return Response({
             'errors': 'sorry article with that slug doesnot exist'
         }, status=status.HTTP_404_NOT_FOUND)
+
+
+class LikeArticleAPIView(generics.GenericAPIView):
```

Comment:
Is it possible to have a single `LikeDislikeApiView` to avoid the repetition? cc: @Karuhanga

Reply:
@malep2007 yes. I think it can be done.
I was trying to avoid requiring the extra work of having to pass a body, but I now realize we could use the same view and just pass a parameter to the `as_view` method.
It does introduce a few inelegant ifs in there though.
I don't see yet how that can be avoided.

Reply:
Just made this change. Removes a lot of the duplication. Thanks @malep2007.